storm-core/src/clj/org/apache/storm/daemon/supervisor.clj

```diff
@@ -1088,8 +1104,26 @@
                                             (str "file:///" storm-log4j2-conf-dir))
                                           storm-log4j2-conf-dir)
                                      file-path-separator "worker.xml")
+
+          cgroup-command (if (conf STORM-CGROUP-ENABLE)
```

Comment:
cgroups is disabled by default (via default.yaml) and the code you referenced is the code that will be executed for "Cluster mode" and not "local mode".